pkg/kubelet/kubelet.go

```diff
@@ -2865,13 +2978,15 @@ func (kl *Kubelet) CheckpointContainer(
 		return fmt.Errorf("container %v not found", containerName)
 	}
 
+	checkpointTime := time.Now().Unix()
+
 	options.Location = filepath.Join(
 		kl.getCheckpointsDir(),
 		fmt.Sprintf(
 			"checkpoint-%s-%s-%s.tar",
 			podFullName,
 			containerName,
-			time.Now().Format(time.RFC3339),
+			time.Unix(checkpointTime, 0).Format(time.RFC3339),
```

Comment:
@adrianreber Can you explain the difference between `time.Now().Format(time.RFC3339)` and `time.Unix(time.Now().Unix(), 0).Format(time.RFC3339)` and why the latter is preferred?

Reply:
I think this is a left-over from changes that happened during review of this PR. Initially I had a json file for each pod/container combination tracking the available checkpoints. That was removed during the review to actually scan the disk instead of using additional metadata (json file). I will take a closer look at this change, if it is still necessary.

Reply:
removed the change again